charts/actions-runner-controller/templates/githubwebhook.ingress.yaml

```
@@ -34,8 +36,15 @@ spec:
           {{- range .paths }}
           - path: {{ .path }}
             backend:
+              {{- if .Capabilities.APIVersions.Has "networking.k8s.io/v1" }}
+              service:
+               name: {{ $fullName }}
+               port:
+                 number: {{ $svcPort }}
+              {{- else -}}
```

This needs to be:

```suggestion
              {{- else }}
```

Otherwise it breaks for any non-`networking.k8s.io/v1`-case like

```
spec:
  rules:
    - host: "example"
      http:
        paths:
          - path: mypath
            backend:serviceName: foo-actions-runner-controller-github-webhook-server
              servicePort: 80
```